libafl/src/mutators/multi.rs

```diff
@@ -207,7 +213,7 @@ where
                 size,
                 target,
                 range,
-                &other.parts()[choice],
+                other.parts()[choice].bytes(),
```

Comment:
Why is this needed? Can't we implement `HasMutatorBytes` for MappedInput<T> where T: HasMutatorBytes?

Reply:
Ah crossover takes a slice now. Fine I guess